src/elements/content-explorer/PreviewDialog.scss

```diff
@@ -0,0 +1,16 @@
+[class^='bp_modal_module_content'].bce-PreviewDialog {
+    position: fixed;
+    top: 0;
+    right: 0;
+    z-index: 1000;
```

Comment:
`z-index` is pretty fragile, id avoid updating if possible. we'd have to test against how it interacts with other floating components like tooltips, dropdowns, selects, datepicker, flyout, annotations, etc